### PR TITLE
fast Versioning

### PR DESCRIPTION
**Pull request by crafter23456**
# Description

Unnecessary to fetch it every startup again, because we stay with 1.8.8 RC2-SNAPSHOT.

# Checklist:

- [x] I have reviewed my code thoroughly.
- [x] I have tested my code.
- [x] My changes generate no new warnings

### CODE REVIEW

File: NachoSpigot-Server/src/main/java/org/bukkit/craftbukkit/util/Versioning.java [hunk 1]
```diff
@@ -1,29 +1,7 @@
...
-                properties.load(stream);
-
-                result = properties.getProperty("version");
-            } catch (IOException ex) {
-                Logger.getLogger(Versioning.class.getName()).log(Level.SEVERE, "Could not get Bukkit version!", ex);
-            }
-        }
-
-        return result;
+        return "1.8.8-R0.2-SNAPSHOT";
```
Comment (Elierrr) [inline NachoSpigot-Server/src/main/java/org/bukkit/craftbukkit/util/Versioning.java:5]: @Lucaskyy Should we finally switch to `1.8.8-R0.1-SNAPSHOT` like upstream?

**Review by Elierrr — APPROVED** [review]

File: NachoSpigot-Server/src/main/java/org/bukkit/craftbukkit/util/Versioning.java (hunk 1 above)
Comment (LewUwU) [reply NachoSpigot-Server/src/main/java/org/bukkit/craftbukkit/util/Versioning.java:5]: R0.1 is the most used

Comment (Lucaskyy) [reply NachoSpigot-Server/src/main/java/org/bukkit/craftbukkit/util/Versioning.java:5]: @crafter23456 ^

**Review by Elierrr — CHANGES_REQUESTED** [review]
Update the version in the pom.xml as well

**Review by Elierrr — APPROVED** [review]

**Review by Lucaskyy — APPROVED** [review]

**Pull request merged**